src/modules/storage/LedgerStorage.ts

```diff
@@ -2342,6 +2533,65 @@ export class LedgerStorage extends Storages
         return this.query(sql, [from, to]);
     }
 
+   /**
+    * Get the transaction count for an address
+    * @returns Returns the Promise. If it is finished successfully the `.then`
+    * of the returned Promise is called with the records
+    * and if an error occurs the `.catch` is called with an error.
+    */
+    public getTxCount(height: Height, address: string): Promise<any[]> 
+    {
+        let sql =
+            `SELECT
+               COUNT(DISTINCT(tx_hash)) AS tx_count
+            FROM
+            (
+                SELECT
+                    T.tx_hash
+                FROM
+                    tx_outputs S
+                    INNER JOIN tx_inputs I ON (I.utxo = S.utxo_key)
+                    INNER JOIN transactions T ON (T.tx_hash = I.tx_hash)
+                WHERE
+                    T.block_height = ?
+                    AND S.address = ?
+                GROUP BY T.tx_hash, S.address
+                
+                UNION ALL
+                
+                SELECT
+                    T.tx_hash
+                FROM
+                    tx_outputs O
+                    INNER JOIN transactions T ON (T.tx_hash = O.tx_hash)
+                WHERE
+                    T.block_height = ?
+                    AND O.address = ?
+                GROUP BY T.tx_hash, O.address
+            ) Tx;`;
+        return this.query(sql, [height.value.toString(), address, height.value.toString(), address]);
+    }
+
+    /**
+     * Get BOA Holder List.
+     * @param limit Maximum record count that can be obtained from one query
+     * @param page The number on the page, this value begins with 1
+     * @returns returns the Promise with requested data
+     * and if an error occurs the .catch is called with an error. 
+     */
+    public getBOAHolders (limit: number, page: number): Promise<any>
+    {
+        let sql = `
+            SELECT 
+	            address, tx_count, total_received, total_sent, 
+	            total_reward, total_frozen, total_spendable, total_balance
+            FROM
+                accounts
+            ORDER BY total_balance DESC
+            LIMIT ? OFFSET ?`
```

Comment:
```suggestion
            SELECT 
	            address, tx_count, total_received, total_sent, 
	            total_reward, total_frozen, total_spendable, total_balance
            FROM
                accounts
            ORDER BY total_balance DESC, address ASC
            LIMIT ? OFFSET ?`
```
There can be multiple addresses with the same balance.
Please add the function of sorting the addresses because the same address can come out across multiple pages.
Also, there is a possibility that the test code may not pass the test in some cases because there is a possibility of inconsistent results.